tests/integration/components/ember-popper/action-test.js

```diff
@@ -2,51 +2,48 @@ import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 import { triggerEvent } from 'ember-native-dom-helpers';
 import wait from 'ember-test-helpers/wait';
-import hasEmberVersion from 'ember-test-helpers/has-ember-version';
 
-if (hasEmberVersion(1, 13)) {
-  moduleForComponent('ember-popper', 'Integration | Component | actions', {
-    integration: true
-  });
+moduleForComponent('ember-popper', 'Integration | Component | actions', {
+  integration: true
+});
 
-  test('it calls onCreate', function(assert) {
-    assert.expect(2);
-    let called = 0;
-    const action = (data) => {
-      called++;
-      assert.ok(data && data.instance, 'onCreate action is called with dataObject');
-    };
-    this.on('create', action);
-    this.render(hbs`
+test('it calls onCreate', function(assert) {
+  assert.expect(2);
+  let called = 0;
+  const action = (data) => {
+    called++;
+    assert.ok(data && data.instance, 'onCreate action is called with dataObject');
+  };
+  this.on('create', action);
+  this.render(hbs`
     {{#ember-popper onCreate=(action "create")}}
       template block text
     {{/ember-popper}}
   `);
 
-    return wait()
-      .then(() => assert.equal(called, 1, 'onCreate action has been called'));
-  });
+  return wait()
+    .then(() => assert.equal(called, 1, 'onCreate action has been called'));
+});
 
-  test('it calls onUpdate', function(assert) {
-    assert.expect(2);
-    let called = 0;
-    const action = (data) => {
-      called++;
-      assert.ok(data && data.instance, 'onUpdate action is called with dataObject');
-    };
-    this.on('update', action);
-    this.render(hbs`
+test('it calls onUpdate', function(assert) {
+  assert.expect(2);
+  let called = 0;
+  const action = (data) => {
+    called++;
+    assert.ok(data && data.instance, 'onUpdate action is called with dataObject');
+  };
+  this.on('update', action);
+  this.render(hbs`
     {{#ember-popper onUpdate=(action "update")}}
       template block text
     {{/ember-popper}}
   `);
 
-    return wait()
-      .then(() => triggerEvent(document.querySelector('body'), 'scroll'))
-      .then(wait)
-      .then(() => {
-        assert.equal(called, 1, 'onUpdate action has been called after event');
-        return wait();
-      });
-  });
-}
+  return wait()
+    .then(() => triggerEvent(document.querySelector('body'), 'scroll'))
+    .then(() => wait())
```

Comment:
FYI: this change (`wait` -> `() => wait()`) is unrelated to my changes but was required, as the latest `ember-test-helpers` was causing this error when some argument (here the resolved promise) is passed to `wait`: https://travis-ci.org/kybishop/ember-popper/jobs/328856339#L547.

See https://github.com/emberjs/ember-test-helpers/blob/9eb3e32f8340d01704e950f317b282f07073f1e9/addon-test-support/%40ember/test-helpers/settled.js#L169-L174

Don't know if this should be regarded a regression of `ember-test-helpers/wait` or not? 🤔  /cc @rwjblue

Reply:
Ya, can you report over there so we can dig into it?

Reply:
https://github.com/emberjs/ember-test-helpers/issues/300